third_party/move/move-prover/src/lib.rs

```diff
@@ -83,7 +83,7 @@ pub fn run_move_prover_v2<W: WriteColor>(
         experiments: vec![],
         experiment_cache: Default::default(),
         sources: cloned_options.move_sources,
-        sources_deps: cloned_options.move_deps,
+        sources_deps: vec![],
```

Comment:
What is this source_deps? Should have never been added to the prover, lets remove this later.

Reply:
This field is from`move_compiler_v2::Options`.